### PR TITLE
Update Finnish translation to Feb. 2023

### PR DESCRIPTION
**Pull request by tnorr**

### CODE REVIEW

File: finnish/wca-regulations.md [hunk 1]
```diff
@@ -227,9 +241,10 @@ Huomautus: Koska artikloille ja säännöille ei määrätä uusia numeroita kun
...
-    - 9f2) Kaikki yli 10 minuutin ajat ja (typistetyt) keskiarvot sekä kaikki ajat 3x3x3 multisokko -tuloksissa mitataan ja pyöristetään lähimpään sekuntiin (esim. x.49 on x, x.50 on x+1).
+    - 9f1) Kaikki alle 10 minuutin ajat, poislukien 3x3x3 multisokko, mitataan ja katkaistaan sekunnin sadasosan tarkkuudelle. Kaikki alle 10 minuutin (typistetyt) keskiarvot mitataan ja pyöristetään lähimpään sekunnin sadasosaan.
+    - 9f2) Kaikki yli 10 minuutin ajat ja (typistetyt) keskiarvot sekä kaikki ajat 3x3x3 multisokko -tuloksissa mitataan ja pyöristetään lähimpään sekuntiin (esim. X.49 on X, X.50 on X+1).
     - 9f4) Ratkaisuyrityksen tulos on DNF (Did Not Finish), jos ratkaisu on hylätty tai ratkaisematon/keskeneräinen.
+	    9f4a) Jos kilpailija rikkoo jotakin sääntöä niin selkeästi, että ratkaisuyritys hylätään varmasti, tuomarin pitäisi keskeyttää ratkaisuyritys heti. Jos tuomari on epävarma tai jos keskeyttäminen voisi olla häiritsevää (esim. koska ylimääräinen ratkaisuyritys voisi viivästyttää kilpailua), tuomarin pitäisi kysyä WCA-delegaatin näkemystä.
```
Comment (hanwu85) [inline finnish/wca-regulations.md:247]:
```suggestion
        - 9f4a) Jos kilpailija rikkoo jotakin sääntöä niin selkeästi, että ratkaisuyritys hylätään varmasti, tuomarin pitäisi keskeyttää ratkaisuyritys heti. Jos tuomari on epävarma tai jos keskeyttäminen voisi olla häiritsevää (esim. koska ylimääräinen ratkaisuyritys voisi viivästyttää kilpailua), tuomarin pitäisi kysyä WCA-delegaatin näkemystä.
```

**Comment by Nanush7** [comment]
Thank you very much for the update, Timo! :)

Do you mind waiting for the new official version to be released (in August), so you can update this one to the latest version? The proofreading/reviewing process  could take some weeks, so I think it makes more sense to wait until August, and publish an up-to-date translation.

**Comment by Nanush7** [comment]
@tnorr ?

**Comment by tnorr** [comment]
Oh, sorry for missing the reply! Sounds good, let's merge only once the new version is official.

**Comment by tnorr** [comment]
I'll update this to the August version soon!

**Comment by Nanush7** [comment]
Hi @tnorr, how do you want to proceed with this translation? :)

**Comment by Nanush7** [comment]
Please let us know if you want to re-open this! ;)

**Pull request closed without merging**